birdhouse/scripts/backup-jupyterhub-notebooks.sh

```diff
@@ -6,7 +6,7 @@ if [ -z "$BACKUP_OUT_DIR" ]; then
 fi
 
 if [ -z "$JUPYTERHUB_USER_DATA_DIR" ]; then
-    JUPYTERHUB_USER_DATA_DIR=/data/jupyterhub_user_data
+    JUPYTERHUB_USER_DATA_DIR=${DATA_PERSIST_ROOT:-/data}/jupyterhub_user_data
```

Comment:
Can't remember, but if this script do not source `defualt.env` and/or `env.local` then yes you should duplicate the default value here.  Otherwise please avoid duplication when possible.

Reply:
I'm not sure either because its not a .template. 
That's why I don't take the chance.